Calculator/Calculator/Model/CalculatorItemQueue.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  CalculatorItemQueue.swift
+//  Calculator
+//
+//  Created by Baem on 2022/09/20.
+//
+
+struct CalculatorItemQueue {
+    private var queue: [CalculatorItem] = []
+    var isEmpty: Bool {
+        return queue.isEmpty ? true : false
+    }
+    var count: Int {
+        return queue.count
+    }
+    
+    mutating func enqueue(_ element: CalculatorItem) {
+        queue.append(element)
+    }
+    
+    mutating func dequeue() -> CalculatorItem? {
+        if queue.isEmpty {
+            return nil
+        }
+        
+        return queue.removeFirst()
+    }
+    
+    func peek() -> CalculatorItem? {
+        return queue.first ?? nil
+    }
```

Comment:
이 메소드도 필요한 이유를 잘 모르겠습니다. 테스트를 위해 추가한 기능일까요?

추가적으로 `first`의 반환형이 옵셔널이니까 아래와 같이 바꾸는게 좋아보여요.
```suggestion
    func peek() -> CalculatorItem? {
        return queue.first
    }
```

Reply:
isEmpty와 first에 대해 정확하게 모르고 썼다는 생각이 드네요.. 
앞으로는 타입확인하면서 더 정확하게 쓰도록 하겠습니다 감사합니다!